Nemt/nemt.cpp

```diff
@@ -15,44 +15,46 @@ Nemt::Nemt(int cl) {
     counter = 0;
 }
 
-int Nemt::giveX(int x) {
-    giveV(x);
+int Nemt::give_x(int x) {
+    give_v(x);
     return x_point_goal;
 }
 
-int Nemt::giveLeft(int x) {
-    giveV(x);
+int Nemt::give_left(int x) {
+    give_v(x);
     return left_point_goal;
 }
 
-int Nemt::giveRight(int x) {
-    giveV(x);
+int Nemt::give_right(int x) {
+    give_v(x);
     return right_point_goal;
 }
 
-void Nemt::giveV(int block_color) {
+void Nemt::give_v(int block_color) {
     blocks = pixy.getBlocks();
-    if(blocks) // if we see at least one objects we can be happy 
-    {
-        for (int i = 0; i < blocks; i++) // runnig through all objects 
-        {
-            // if object i == goal_color and is bigger that captured object that it is object with bigger priority
-            if(pixy.blocks[i].signature == block_color &&
-               pixy.blocks[i].width > pixy.blocks[goal_number].width)
-            { 
-                goal_number = i; // we set color of goal
+
+    if (blocks) {
+        for (int i = 0; i < blocks; i++) {
+            if (pixy.blocks[i].signature == block_color &&
+               pixy.blocks[i].width > pixy.blocks[goal_number].width) {
+                goal_number = i;
             }            
-        } 
+        }
 
         x_point_goal = pixy.blocks[goal_number].x;
-        left_point_goal = pixy.blocks[goal_number].x - pixy.blocks[goal_number].width/2; // left pint of goal // lpg = x - width/2    
-        right_point_goal = pixy.blocks[goal_number].x + pixy.blocks[goal_number].width/2; // right point of goal // rpg = x + width/2   
+
+        left_point_goal = pixy.blocks[goal_number].x
+                - pixy.blocks[goal_number].width/2;
```

Comment:
Please align those operators (`-` and `+` on line 50) with `=` signs on previous lines.